presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcPageSink.java

```diff
@@ -83,6 +66,19 @@ public JdbcPageSink(JdbcOutputTableHandle handle, JdbcClient jdbcClient)
         }
 
         columnTypes = handle.getColumnTypes();
+
+        columnWriters = columnTypes.stream()
+                .map(type -> {
+                    WriteFunction writeFunction = jdbcClient.toWriteMapping(type).getWriteFunction();
+                    checkState(
+                            type.getJavaType() == writeFunction.getJavaType(),
+                            "Presto type %s is not compatible with write function %s accepting %s",
+                            type,
+                            writeFunction,
+                            writeFunction.getJavaType());
```

Comment:
`.getName()`

Reply:
"... write function xxx accepting **class** io.airlift.slice.Slice" will be OK.
(also, for primitive types, there is no difference at all)

since this should never reach end user, i prefer brevity of the code